packages/legacy-client/src/client/ontology.ts

```diff
@@ -38,6 +41,10 @@ export class Ontology<O extends OntologyDefinition<any>> {
     return createActionProxy(this.#client);
   }
 
+  get bulkActions(): BulkActions<O> {
```

Comment:
I only added this here, and not in the actual OntologyDefinition since this seems like we're only adding this for 1.1

Reply:
That would be correct, since there is no custom definition for actions vs bulk actions. Only the runtime `client.ontology.` should be showing `bulkActions`.